planck/PLKClojureScriptEngine.m

```diff
@@ -552,6 +558,66 @@ -(void)startInitializationWithSrcPaths:(NSArray*)srcPaths outPath:(NSString*)out
                                          argList:@"path"
                                        inContext:self.context];
         
+
+        [ABYUtils installGlobalFunctionWithBlock:
+         ^JSValueRef(JSContextRef ctx, size_t argc, const JSValueRef argv[]) {
+             
+             if (argc == 1 && JSValueGetType (ctx, argv[0]) == kJSTypeString) {
+
+                 NSString* path = NSStringFromJSValueRef(ctx, argv[0]);
+
+                 NSDictionary* result = [[NSFileManager defaultManager]attributesOfItemAtPath:path error:nil];
```

Comment:
To me it seems like returning `nil` is the way to go, since `NSFileManager attributesOfItemAtPath` returns an empty `NSDictionary`. 
This also plays well with Clojure since `(when (exists? "file") ...)` essentially becomes `(when (fstat "file") ..)`